extensions/sequelize/src/sequelize/sequelize.repository.base.ts

```diff
@@ -686,16 +688,22 @@ export class SequelizeCrudRepository<
    * @param options Sequelize Sync Options
    */
   async syncSequelizeModel(options: SyncOptions = {}) {
-    await this.dataSource.sequelize?.models[this.entityClass.modelName]
-      .sync(options)
-      .catch(console.error);
+    if (!this.dataSource.sequelize) {
+      throw new Error(
+        'Sequelize instance is not attached to the datasource yet.',
+      );
+    }
+    await this.dataSource.sequelize.authenticate();
```

Comment:
Why authenticate ?

Reply:
This is to match the loopback behaviour of checking the db connection on initialization.

Sequelize doesn't by default checks if the connection is okay, until the first query is requested.

Hence the authenticate call is placed there.

Reply:
ok. Makes sense